GDJS/Runtime/runtimeobject.js

```diff
@@ -104,6 +169,10 @@ gdjs.RuntimeObject = function(runtimeScene, objectData) {
 
     //Timers:
     if (this._timers === undefined)
+        /**
+         * @type {Hashtable<string, gdjs.Timer>}
```

Comment:
This is what should be the proper type but Hashtable is not a generic type so Typescript will probably yell at this if we add `// @ts-check`. Let's either use Hashtable, or declare a generic type for Hashtable (I hope it can be done and compatible with JSDoc)

Reply:
Well, JSDoc supports generics, I used them, only to realise that typescript doesn't support type checking from JSDocs generics and makes an error for each use of hashtable in the code

Reply:
In this case let's avoid this for now, we can deal with Hashtable in a separate PR if you wish. Maybe take a look at @template: https://www.typescriptlang.org/docs/handbook/type-checking-javascript-files.html#template - not sure this can be used properly though.

Reply:
The link you sent was what I was using as documentation. I think the only way would be to add a typescript type definition instad of JSDoc definitions. You said you don't want them, but that was because we could use JSDoc instead, that also handles documentation. Could we use Typescript typedefs just for that specific case?

Reply:
> Could we use Typescript typedefs just for that specific case

Yes, if it does not break documentation generation.
If this is too painful we'll switch to typedoc (https://typedoc.org/) that should be hopefully 100% compatible with Typescript.

Reply:
Nevermind I am stupid. This would work for typescript but wouldn't allow using Hastable<> syntax in JSDoc comments so I will have to leave that out for the moment.

Reply:
I'm still hoping we can keep JSDoc as much as possible, because it is quite widespread in the community and battle tested :) 
Annoyingly it's rejecting sometimes some type annotations that are too complex (for example, a function type like `(param: number) => string` is not properly parsed).